### ISSUE TITLE
Label customization

### ISSUE THREAD

**Issue by aleclarson** [intuit/auto#218]
Support custom hex colors for labels in the `.autorc` configuration. 😄 🎨

**Comment by aleclarson** [comment]
My suggestion would be to allow the following:

```json
{
  "labels": {
    "major": { "name": "Version: Major", "color": "#000000" }
  }
}
```

The `name` and `color` properties are both optional, but at least one must exist.

**Comment by aleclarson** [comment]
While we're at it, you could allow a `description` property, too!

**Comment by aleclarson** [comment]
And if I could customize the `documentation` label to be `docs` instead, that would be perfect.

**Comment by aleclarson** [comment]
Lastly, we could make the `#` in colors optional, and just prepend it for the user.

**Comment by aleclarson** [comment]
And maybe a value of `false` should prevent the `auto-release` from creating that label. My use case would be to prevent the "release" label from being created.

**Comment by aleclarson** [comment]
You should be able to add custom labels, too.

And an `update-labels` command might be warranted?

**Comment by hipstersmoothie** [comment]
`release` should only be created if you've set up `onlyPublishWithReleaseLabel` mode. If it isn't thats a bug

**Comment by hipstersmoothie** [comment]
so you want auto to add labels that aren't use for versioning or the changelog also?

**Comment by aleclarson** [comment]
@hipstersmoothie That's what happened with me then, but if I change that option, the label remains.

> so you want auto to add labels that aren't use for versioning or the changelog also?

It seems like a good place to consolidate all labels of a repository and provides a nice medium for sharing labels between repositories.

**Comment by hipstersmoothie** [comment]
:shipit: Issue was released in 10.0.0 :rocket:

**Comment by hipstersmoothie** [comment]
:rocket: Issue was released in 10.0.0 :rocket:
